WordPress/Classes/Models/Blog.h

```diff
@@ -67,6 +73,7 @@ typedef NS_ENUM(NSUInteger, BlogFeature) {
 @property (nonatomic, strong, readwrite) NSString *icon;
 @property (nonatomic, strong, readwrite) NSNumber *defaultCategoryID;
 @property (nonatomic, strong, readwrite) NSString *defaultPostFormat;
+@property (nonatomic, strong, readwrite) NSNumber *privacy;
```

Comment:
I'm tempted to say we should maybe hide this property and only provide a convenience getter/setter which maps the enum to the NSNumber, using the property willAccess, etc methods. NSNumber *privacy doesn't provide enough insight into what this field is/does/contains in my opinion.